apps/explorer/src/components/validator/ValidatorMeta.tsx

```diff
@@ -51,7 +138,7 @@ export function ValidatorMeta({ validatorData }: ValidatorMetaProps): JSX.Elemen
                         </div>
                     </div>
                     <div className="flex w-1/2 flex-col gap-y-md">
-                        <span className="text-label-lg text-neutral-40 dark:text-neutral-60">
+                        <span className="text-label-lg text-neutral-40 dark:text-neutral-6">
```

Comment:
revert this please :pray: 
```suggestion
                        <span className="text-label-lg text-neutral-40 dark:text-neutral-60">
```